README.md

```diff
@@ -4,7 +4,7 @@ Initially copied from Illinois Sunshine.
 
 ## Setup
 
-**Install OS level dependencies:** 
+**Install OS level dependencies:**
 
 * Python 3.4
 * PostgreSQL 9.4 +
```

Comment:
I mean, if you're gonna fix the README anyways... maybe you can change this PostgreSQL requirement to MySQL for us too 😉

Reply:
lolz, I changed the first thing my eye caught.